integration/ledger/snapshot_test.go

```diff
@@ -1038,59 +1036,69 @@ func invokeWithoutPassingOrdererEndPoint(n *nwo.Network, peer *nwo.Peer, channel
 
 // commitTx commits a transaction for a given transaction envelope
 func commitTx(n *nwo.Network, orderer *nwo.Orderer, peer *nwo.Peer, channelID string, env *cb.Envelope, txid string) error {
```

Comment:
I like how you changed "env" to "tx" in the other CommitTx() further down, could you do the same here for consistency.